File: java_backup/my java/siddhartha/Siddhartha/Prime.java


public class Prime
{public static void main(int n)
{int c=0;
for(int i=1;i<=n;i++)
{if (n%i==0)
{c++;
}
}
if (c==2)
{System.out.println("prime");
}
else
{System.out.println("not prime");
}
}
}
















    

File: 3rd_sem_sxc/SEM==3/BEFORE_SEM/prime.pl
system("clear");
printf("\n Enter the upper bound of the prime range : \n");
$n=<STDIN>;



for($i=2;$i<=$n;$i++)
{
	$check=&isPrime($i);
	if($check==1)
	{
		printf("%d\t",i);
	}	
	#printf("i = %d, check = %d",$i,$check);

}
sub isPrime
{
	local($x)=@_;
	$chk=0
	for($j=2;$j<$x;$j++)
	{
		if($x%$j==0)
		{
			$chk=1
		}
	
	}
	if($chk==1)
	{
		0;
	}
	else
	{
		1;	
	}
}

